bindings/ios/MEGASdk.mm

```diff
@@ -1497,6 +1497,26 @@ - (void)killSession:(uint64_t)sessionHandle {
     self.megaApi->killSession(sessionHandle);
 }
 
+- (nonnull NSDate *)overquotaDeadlineDate {
+    return [[NSDate alloc] initWithTimeIntervalSince1970:self.megaApi->getOverquotaDeadlineTs()];
+}
+
+- (nonnull NSArray<NSDate *> *)overquotaWarningDateList {
+    MegaIntegerList *warningTimeIntervalList = self.megaApi->getOverquotaWarningsTs();
+
+    int sizeOfWarningTimestamps = warningTimeIntervalList->size();
+    if (sizeOfWarningTimestamps == 0) {
+        return [[NSMutableArray alloc] initWithCapacity:0];
```

Comment:
we don't need to check 0 here, the for loop can handle 0 case. make the code simpler 😉